assets/xml/objects/object_keikoku_demo.xml

```diff
@@ -7,21 +7,21 @@
 
         <!-- DmOpstage draws both an empty XLU and non-empty OPA at the same time, each version has its own empty DL instead of reusing. -->
         <DList Name="gKeikokuDemoFloorEmptyDL" Offset="0x970" />
-        <DList Name="gKeikokuDemoFloorDL" Offset="0x978" />
+        <DList Name="gKeikokuDemoFloorDL" Offset="0x978" /> <!-- Original name is "daichi_model" -->
 
         <!-- DmOpstage type DMOPSTAGE_TYPE_GROUND textures -->
         <Texture Name="gKeikokuDemoYellowFlowersOnGrassTex" OutName="yellow_flowers_on_grass" Format="rgba16" Width="32" Height="32" Offset="0xC50" />
         <Texture Name="gKeikokuDemoFallLeavesOnGrassTex" OutName="fall_leaves_on_grass" Format="rgba16" Width="32" Height="32" Offset="0x1450" />
 
         <!-- DmOpstage type DMOPSTAGE_TYPE_GROUND Dyna Collider -->
-        <Collision Name="gKeikokuDemoFloorColliderHeader" Offset="0x1C98" />
+        <Collision Name="gKeikokuDemoFloorCol" Offset="0x1C98" /> <!-- Original name is "daichi_bgdatainfo" -->
 
         <DList Name="gKeikokuDemoTallTreeWithRootBaseEmptyDL" Offset="0x2870" />
-        <DList Name="gKeikokuDemoTallTreeWithRootBaseDL" Offset="0x2878" />
+        <DList Name="gKeikokuDemoTallTreeWithRootBaseDL" Offset="0x2878" /> <!-- Original name is "lost_woods_wood1_model" -->
         <DList Name="gKeikokuDemoTallTreeCutEmptyDL" Offset="0x3060" />
-        <DList Name="gKeikokuDemoTallTreeCutDL" Offset="0x3068" />
+        <DList Name="gKeikokuDemoTallTreeCutDL" Offset="0x3068" /> <!-- Original name is "lost_woods_wood2_model" -->
         <DList Name="gKeikokuDemoTallTreeStraightEmptyDL" Offset="0x3720" />
-        <DList Name="gKeikokuDemoTallTreeStraightDL" Offset="0x3728" />
+        <DList Name="gKeikokuDemoTallTreeStraightDL" Offset="0x3728" /> <!-- Original name is "lost_woods_wood3_model" -->
```

Comment:
`Woods_wood2_` squint your eyes a bit and its almost looks deliberately obfuscated, man...